docs/source/deployment/azure-functions.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Deploying with Azure Functions
+sidebar_title: Azure Functions
+description: Deploying your GraphQL server to Azure Functions
+---
+
+import {
+  ExpansionPanel,
+} from 'gatsby-theme-apollo-docs/src/components/expansion-panel';
+
+This tutorial walks through setting up an example project using the [Azure Functions integration](https://www.npmjs.com/package/apollo-server-azure-functions) for Apollo Server.
+
+We created this tutorial using a Linux environment. Some commands might differ if you use a Windows environment; [please refer to the Azure documentation for more details](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli).
+
+Note that `apollo-server-azure-functions` does not enable you to add arbitrary middleware to your web server (though you can manually wrap the handler returned by `createHandler` in your own handler).
+
+## Prerequisites
+
+You must do following before proceeding:
+
+- Setup an [Azure](https://azure.com) account
+- Install the [Azure Functions Core Tools CLI version 3.x](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#v2)
+- Install the [Azure CLI 2.x](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+
+## Setting up the project
+
+### Creating a new Function
+
+We'll begin by creating a new project to test things out locally before deploying to Azure:
+
+```shell
+func init apollo-example --worker-runtime node
+cd apollo-example
+func new --template "Http Trigger" --name graphql
+```
+
+Next, run the `func host start` command. You should see something like the following output to your terminal:
+
+```shell
+Azure Functions Core Tools
+Core Tools Version:       3.0.3477 Commit hash: 5fbb9a76fc00e4168f2cc90d6ff0afe5373afc6d  (64-bit)
+Function Runtime Version: 3.0.15584.0
+
+[2021-06-30T19:22:21.077Z] Worker process started and initialized.
+
+Functions:
+
+	graphql: [GET,POST] http://localhost:7071/api/graphql
+
+For detailed output, run func with --verbose flag.
+```
+
+If you navigate to [http://localhost:7071/api/graphql?name=Apollo](http://localhost:7071/api/graphql?name=Apollo) in your browser and will now see the text: 'Hello Apollo.'
+
+<ExpansionPanel title="Run into a problem?">
+
+If you see an error message mentioning "Incompatible Node.js version", you should double-check that the version of `node` on your `PATH` is a version [supported by Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=in-process%2Cv4&pivots=programming-language-javascript). If your node version is not supported, you can use a tool like [nvm](https://github.com/nvm-sh/nvm) to use a different node version.
```

Comment:
"you should double-check" -> "confirm"

`node` -> Node.js (in all spots here)

"to use a different node version" -> "to use a different version"